application-admintools-ui/src/main/resources/AdminTools/Code/Configuration.xml

```diff
@@ -334,7 +334,15 @@
       <categoryIcon/>
     </property>
     <property>
-      <codeToExecute/>
+      <codeToExecute>#set ($adminToolsConfigClassName = 'AdminTools.Code.ConfigurationClass')
+#set ($classExists = $xwiki.exists($adminToolsConfigClassName))
+#if (!$classExists)
+  #set($infoMessage = "The configuration options for Admin Tools Pro are unavailable because the application has been uninstalled.&lt;br&gt;To remove Admin Tools Pro from administration configuration, please &lt;a href=$xwiki.getURL('AdminTools.Code.Configuration', 'delete')&gt;&lt;b&gt;delete this page&lt;/b&gt;&lt;/a&gt;. Note that deleting the page will permanently erase any custom configurations made on Admin Tools Pro configuration.")
+
+{{html clean=false}}
+  #info($infoMessage)
```

Comment:
Why not using the [info macro](https://extensions.xwiki.org/xwiki/bin/view/Extension/Info%20Macro)? It easier to read / write than having all in a single (long) line. (since you cannot use translations)
For example:
```
{{info}}
{{html}}
<p>The configuration options for Admin Tools Pro are unavailable because the application has been uninstalled.
</p>
</p>
  To remove Admin Tools Pro from administration configuration, please 
  <a href=$xwiki.getURL('AdminTools.Code.Configuration', 'delete')><b>delete this page</b></a>. 
  Note that deleting the page will <b>permanently erase</b> any custom configurations made on 
  Admin Tools Pro configuration.
</p>
{{/html}}
{{/info}}
```